### PR TITLE
Stats: Changed the docs of crv_types.py

### PR DESCRIPTION
**Pull request by supreet11agrawal**
Restored the changes in docs which were made in an earlier PR.

#### Release Notes

NO ENTRY

### CODE REVIEW

File: sympy/stats/crv_types.py [hunk 1]
```diff
@@ -108,7 +108,7 @@
 
 def ContinuousRV(symbol, density, set=Interval(-oo, oo)):
     """
-    Creates a Continuous Random Variable given the following:
+    Create a Continuous Random Variable given the following:
```
Comment (oscargus) [inline sympy/stats/crv_types.py]: I've seen that an s is used in some documentation, but I do not have any strong opinions. However, maybe you would like to change to "continuous random variable" as in most cases below.

File: sympy/stats/crv_types.py [hunk 2]
```diff
@@ -175,9 +175,9 @@ def _cdf(self, x):
 
 def Arcsin(name, a=0, b=1):
     r"""
-    Creates a Continuous Random Variable with Arcsine distribution.
+    Create a Continuous Random Variable with an arcsin distribution.
```
Comment (oscargus) [inline sympy/stats/crv_types.py]: Here as well.

File: sympy/stats/crv_types.py [hunk 3]
```diff
@@ -253,7 +253,7 @@ def _moment_generating_function(self, t):
 
 def Benini(name, alpha, beta, sigma):
     r"""
-    Creates a Continuous Random Variable with Benini distribution.
+    Create a Continuous Random Variable with a Benini distribution.
```
Comment (oscargus) [inline sympy/stats/crv_types.py]: And here.

File: sympy/stats/crv_types.py [hunk 4]
```diff
@@ -342,7 +342,7 @@ def _moment_generating_function(self, t):
 
 def Beta(name, alpha, beta):
     r"""
-    Creates a Continuous Random Variable with Beta distribution.
+    Create a Continuous Random Variable with a Beta distribution.
```
Comment (oscargus) [inline sympy/stats/crv_types.py]: And here.

**Comment by supreet11agrawal** [comment]
> Good idea! However, for consistency I believe that it should be changed to "continuous random variable" everywhere. Hence, I cancel the tests for now.

Yes, that can be done. However, I would like to wait for @Upabjojr's reply.

File: sympy/stats/crv_types.py (hunk 1 above)
Comment (jksuom) [reply sympy/stats/crv_types.py]: This was changed a couple of weeks ago. I think it is proper to return the old form recommended by PEP 257:
> The docstring is a phrase ending in a period. It prescribes the function or method's effect as a command ("Do this", "Return that"), not as a description; e.g. don't write "Returns the pathname ...".

File: sympy/stats/crv_types.py (hunk 2 above)
Comment (Upabjojr) [reply sympy/stats/crv_types.py]: Are you human or just a bot?

**Pull request merged**